envs/example.env

```diff
@@ -37,4 +37,8 @@ DEBUG="True" # Set DEBUG=True for Local dev and Development, not Staging or Live
 
 DJANGO_STARTUP_COMMAND="python manage.py runserver 0.0.0.0:8008" # for local development with auto-reload
 # DJANGO_STARTUP_COMMAND="gunicorn --bind=0.0.0.0:8000 --timeout 600 rcpch-audit-engine.wsgi" # for live deployment
-NPDA_SECRET_KEY=##########
\ No newline at end of file
+NPDA_SECRET_KEY=##########
+
+# FOR LOCAL DEVELOPMENT
+LOCAL_DEV_ADMIN_EMAIL="incubator@rcpch.ac.uk"
+LOCAL_DEV_ADMIN_PASSWORD="BR#GbdPH5j#utYShr&%QUB$*BS#YbLA8w6QxN8H4"
```

Comment:
Minor: let's make this an obviously insecure string like "devpassword" just so it doesn't look at a glance like a real password has been committed to the repo

Reply:
I agree. I will make that change. Otherwise looks good to me. Also, in a separate commit elsewhere I have corrected the env in settings.py that pulls in the secret from .env also so should not have to login each autoreload